libraries/libstratosphere/include/stratosphere/sm/sm_types.hpp

```diff
@@ -40,8 +40,8 @@ namespace ams::sm {
             return out;
         }
 
-        static constexpr ServiceName Encode(const char *name) {
-            return Encode(name, std::strlen(name));
```

Comment:
Wonder if we should just add e.g. util::Strlen. We have (as Nintendo does) Strnlen, Strlcpy...ah well, this is probably fine.

Reply:
(no need to make changes here).